standard/coordinates.md

```diff
@@ -0,0 +1,412 @@
+```
+
+%-----------------------------------------------------------------------------
+
+Assuming for the moment that there are no [Fiducial](#s:fiducial) elements present,
+the construction of a tracking
+branch coordinate system starts at the [BeginningEle](#s:begin.ele) element at the start of a
+branch. If the branch is a [root](#s:lattice.def) branch, the orientation of the beginning
+element within the [floor coordinate system](#s:coords) can be fixed by setting 
+[FloorOrientation](#s:orientationition.g) parameters in the `BeginningEle` element.
+If the branch is not a `root` branch, the position
+of the beginning element is determined by the position of the `Fork` element
+from which the branch forks from. The default value of {math}`s` at the `BeginningEle` element is zero
+for both root and non-root branches.
+
+Once the beginning element in a branch is positioned, succeeding elements are concatenated together
+to form the branch coordinate system. All elements have an "`upstream`" and a "`downstream`"
+end as shown in [](#f:patch.between)A. 
+The `downstream` end of an element is always farther (at greater {math}`s`-position) 
+from the beginning element than the `upstream` end of the element. Normally, particles will travel
+in the {math}`+s` direction, so particles will enter an element at the upstream end and exit at the
+downstream end.
+
+If there are `Fiducial` elements, the branch coordinates are constructed beginning at these
+elements working both forwards and backwards along the branch. 
+If there are multiple `Fiducial` elements in a branch, there must be a flexible [Patch](#s:patch)
+element between them.
+
+If an element is not [reversed](#s:ele.reverse),
+the element's `upstream` end is the same as the element's `entrance` end 
+([](#f:ele.coord.frame)) and the `downstream` end is the same 
+as the element's `exit`. If the element is reversed, the `entrance` and `exit` ends are switched.
+That is, for a `reversed` element, particles traveling downstream will
+enter at the element's `exit` end and will exit at the `entrance` end.
+
+The procedure to connect elements together to form the branch coordinates is to ignore 
+`element body` alignment shifts and mate the `downstream` reference frame of
+the element with the `upstream` reference frame of the next element in
+the branch so that the {math}`(x,y,z)` coordinate axes coincide.
+If there are body alignment shifts, the `entrance` and `exit` frames will move with the element. 
+However, this does not affect the branch coordinate system.
+This is illustrated in [](#f:patch.between). [](#f:patch.between)A shows the general situation
+with the downstream frame of element `A` mated to the upstream frame of element `B`.
+The {math}`(x,z)` coordinates are drawn at the entrance end of the elements and {math}`z` will 
+always point towards the element's exit end.
+[](#f:patch.between)B shows a line
+with an unreversed drift named `dft` connected to an unreversed bend named
```

Comment:
As above - maybe we can use a word other than "unreversed"?

Reply:
By "unreversed", you just mean the normal or default operation.  I don't think any modifier is necessary, or at most something like "nominal".  What do you think?

Reply:
How about using: "normal (unreversed)" ?